Testscripts/Linux/ntttcp_monitor_throughput.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# This program is distributed in the hope that it will be useful,
```

Comment:
what is it? GPL?

Reply:
what do you mean? please clarify. The is a common one as other existing scripts in this repo.  just updated the bottom line.

Reply:
GPL is added to Apache 2 License here. CELA mostly recommends choosing one license unless it explicitly requires. Can you clarify we need both or a single?